Source/ACE/Entity/Player.cs

```diff
@@ -2456,6 +2473,30 @@ public void HandleActionDropItem(ObjectGuid itemGuid)
 
             dropChain.EnqueueChain();
         }
+        
+        public void HandleActionUseOnTarget(ObjectGuid sourceObjectId, ObjectGuid targetObjectId)
+        {
+            CurrentLandblock.EnqueueAction(new ActionEventDelegate(() =>
```

Comment:
This should be an action chain on the player, not a direct action on the landblock (its actually not safe to access landblock here, because it could be actively being modified while read :-/).  Instead try:

ActionChain chain = new ActionChain(this, () => 
{ 
    ... 
});
chain.EnqueueChain();

Reply:
will do.